src/map/status.cpp

```diff
@@ -5989,6 +5994,12 @@ void status_calc_bl_main(struct block_list& bl, std::bitset<SCB_MAX> flag)
 			matk_max += sd->bonus.ematk;
 		}
 
+		// Flat MATK not visible in status window
+		if (sd != nullptr && sd->bonus.ematk_hidden > 0) {
```

Comment:
we should just add ematk_hidden even if it's 0, because right now it'll skip if it's negative

Reply:
Correct! ematk also need a fix